### PR TITLE
chore: code formated

### PR DESCRIPTION
**Pull request by nathsep9**
## Motivación 
#1

### CODE REVIEW

**Comment by EduardMcfly** [comment]
@nathsep9 Good job my love

**Review by EduardMcfly — COMMENTED** [review]
This is necessary?

File: blueprints/homePage/templates/mainPage.html [hunk 1]
```diff
@@ -197,7 +197,6 @@ <h2 class="section-title text-center"><b>¿Por qué usar Endurist Tracking?</b>
                         </div><!--//quote-source-->
                         <p>Los resultados de los entrenamientos y las competencias ingresados por el usuario al sistema son analizados por el mismo teniendo en cuenta la categoría a la que pertenece, con el fin de brindar un adecuado y pertinente análisis al deportista acerca de sus actividades deportivas.</p>
                     </blockquote>
-
```
Comment (EduardMcfly) [inline blueprints/homePage/templates/mainPage.html:200]: This is necessary?

Comment (nathsep9) [reply blueprints/homePage/templates/mainPage.html:200]: si por tema de formato

**Pull request merged**